samples/apache-others/challenge/Dockerfile

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+FROM kctf-nsjail
+
+RUN apt-get update && apt-get install -yq --no-install-recommends apache2 && rm -rf /var/lib/apt/lists/*
+
+RUN service apache2 start
+
+RUN ln -s /etc/apache2/mods-available/proxy.load /etc/apache2/mods-enabled/
+RUN ln -s /etc/apache2/mods-available/proxy_http.load /etc/apache2/mods-enabled/
+RUN ln -s /config/apache2-nsjail-others.conf /etc/apache2/conf-enabled/
+
+RUN echo nameserver 8.8.8.8 > /chroot/etc/resolv.conf
```

Comment:
this will be in the default chroot image
https://github.com/google/kctf/pull/36/files#diff-5c8633dd2744a34f5544df8c5b42d713R20